site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -56,6 +56,40 @@ The FusionAuth Id of the User that is being linked to the identity provider.
 include::docs/src/json/identity-providers/links/create-link-request.json[]
 ----
 
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Link a User
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/identity-provider/link#
+--
+
+==== Request Body
+
+[.api]
+[field]#identityProviderLink.displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
+A human readable name for this link. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
++
+This value is optional and it will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP.
```

Comment:
```suggestion
It will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP.
```

Reply:
No need to mention it is optional, already handled in the API line.

Reply:
But I don't understand what you mean by "will be set...". What is it set to? Why even have it if it is overwritten all the time?

Reply:
Hmm, I see you were consistent with other parts of this doc. I am still not sure what this means. Do you understand it?

Reply:
> A human-readable name for this link when initially created. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
> 
> After the link is created, and a user first successfully logs in with the IdP they are linked to, this value will typically be set to either an email or username.